builder/bundleset.go

```diff
@@ -128,15 +128,17 @@ type ValidationLevel int
 // Enum of available validation levels
 const (
 	BasicValidation ValidationLevel = iota
+	BasicValidationIgnoreName
 	StrictValidation
 )
 
 func validateBundleFile(filename string, lvl ValidationLevel) error {
 	var errText string
 
-	// Basic Validation
-	if err := validateBundleFilename(filename); err != nil {
-		errText = err.Error() + "\n"
+	if lvl != BasicValidationIgnoreName {
```

Comment:
why not `if lvl == BasicValidation {`? Imagine we add a new level called `OnlyValidateHeader`. In that case it would also `validateBundleFilename` which would not be what we want to happen.

Reply:
`lvl != BasicValidationIgnoreName` was used instead of `lvl == BasicValidation` so that the definition of BasicValidation does not change.

Reply:
Ok, but why do you call it `BasicValidationIgnoreName`? This is becoming a mess with different levels of `BasicValidation`. I think this needs to be rethought before you modify the concept of `BasicValidation` by providing a way to do it except for one of the checks. Next step I see happening is `StrictValidationPlusExtraCheck` or `StrictValidationIgnoreHeader`. If this is re-thought to either 1) have a menu of validation options or 2) have clear validation levels it will be much cleaner and more maintainable.

Reply:
my point is, you might not be modifying the in-code definition of `BasicValidation` but you are overloading the concept of basic validation. It ends up being confusing and takes a step away from maintainable code. Not claiming the code was particularly maintainable before, but we should takes steps in the right direction.